EnglishFilter/sections/specific.txt

```diff
@@ -306,6 +306,7 @@ $third-party,script,domain=readcomiconline.li|filmyzilla.autos|gaybeeg.info|haca
 kdnuggets.com##div[id^="kdnug-"]
 kdnuggets.com##table li.li-has-thumb + li:not([class])
 pastebin.com##div[style^="padding-bottom:"] div[style*="color:"][style*="font-size:"]
+yugen.to##iframe[style^="width: 100%"][style*="height:"][style*="opacity:"][style*="z-index:"]
```

Comment:
Please take a look to HTML.
If a scriptlet does not help on ios/safari, you rule can be usen for these platforms.